docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -492,6 +492,86 @@ Create a directory at a specific path:
 - **name:** name of the directory
 - **path:** specific path where user wants to create the directory
 
+### Create File
+
+Create a file at a specific path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/touch
+    
+- **name:** name of the file
+- **path:** specific path where user wants to create the file
+
+### Rename
+
+Rename a file or directory:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/rename
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** path after renaming the selected file or directory
+
+### Move
+
+Move a file or directory to a destination path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/move
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** target path for moving the selected file or directory
+
+### Copy
+
+Copy a file or directory to a destination path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/copy
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** target path for copying the selected file or directory
+
+**Note:** For **Apache Ozone** filesystem, copy operation returns a string of skipped files if their size is greater than configured chunk size.
```

Comment:
On the Apache Ozone filesystem, the copy operation returns a string of skipped files if their size is greater than the configured chunk size.